main/course_home/activity.php

```diff
@@ -52,6 +52,12 @@
 		<div class="alert alert-success" id="id_confirmation_message" style="display:none"></div>
 	</div>';
     $content .= $pluginExtra;
+    //show/hide all tools
+    $content .= '<div class="btn-group pull-right">'.
+        '<a class="hidden invisible-all show-hide-all-tools" href="javascript:void(0);"><img  src="'.api_get_path(WEB_IMG_PATH).'invisible.png" alt="'.get_lang('Deactivate', '').'" title="'.get_lang('Deactivate', '').'"></a>'.
+        '<a class="hidden visible-all  show-hide-all-tools" href="javascript:void(0);"><img  src="'.api_get_path(WEB_IMG_PATH).'visible.png" alt="'.get_lang('Activate', '').'" title="'.get_lang('Activate', '').'"></a>'.
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 219 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5714830)